### DISCUSSION TITLE
Issue with screen in Calibration

### DISCUSSION THREAD

**Discussion by Esla2** [sn4k3/UVtools#400] [Q&A] [answered]
Hi All,
At first Thank you for your UVtools.
My issue is all windows in calibration is too large, i can't see all the items in it. And when i resize it it's not display correctly.
Pic attached : Exposure_time_1 : first time i open the window
Exposure_time_2 : when i resize it.
I don't see if i missed thinks when i install it.
![Exposure_time_1](https://user-images.githubusercontent.com/76696830/148934526-d2da445b-7838-4874-a249-b8c28bd9e60e.jpg)
![Exposure_time_2](https://user-images.githubusercontent.com/76696830/148934551-bcc8d24f-f5f9-46ed-b134-d9e559ca71ef.jpg)
![About](https://user-images.githubusercontent.com/76696830/148934572-af4b9abb-63d2-4103-84cc-56c894129af8.jpg)

Thank you for your help.

**Comment by sn4k3** [comment, marked as answer]
Your screen resolution is too low for now-a-days... UVtools was designed with a minimum resolution in mind and this calibration windows are big compared to others tools.
I also guess that you unlocked the window resize on UVtools settings right?

https://github.com/sn4k3/UVtools#requirements
> 1920 x 1080 @ 100% scale as minimum resolution

While i state 1920 x 1080 as minimum, it can go a little lower. The true is that run fine on 720p for example, but everything will be crushed.

However i replicated your resolution and i got a little bit different result:

Without resize:
![UVtools_2022-01-11_12-43-43](https://user-images.githubusercontent.com/113281/148944798-bafeac16-c424-4b42-b04a-e577b768b577.png)

With resizing:
![UVtools_2022-01-11_12-45-46](https://user-images.githubusercontent.com/113281/148945014-79b93a89-6a66-4b97-8309-092572cb24de.png)

The only difference is that the right image preview is showing on my.
On both cases, you can use the mouse scroll wheel to navigate up and down, every important controls are visible, you will only loss the preview image or half of it.

When you resize the window, the scroll bar gets visible and you can use it too.

The only bugs i see here is the profiles text box
And the impossibility to maximize due windows size constrains

You should consider upgrade your monitor, not because of UVtools but to give a quality upgrade to yourself. 
If eyesight is a problem, get something like 27" 1920x1080 (FullHD) and never less

**Comment by Esla2** [comment]
Thank you for quick replay.
And yes my monitor is old, and it is in my list for change.
anyway thank you for UVtools.
Es
